docs_src/general/ServicePorts.md

```diff
@@ -31,6 +33,8 @@ The following tables (organized by type of service) capture the default service
     |device-modbus	|59901|
     |device-bacnet  |59980|
     |device-mqtt	|59982|
+    |device-usb-camera |59983
+    |device-onvif-camera |59984
```

Comment:
missing bar `|`

Reply:
The outer lines are optional, but good for consistency.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#tables